sdk/src/main/java/io/radar/sdk/RadarSettings.kt

```diff
@@ -178,6 +179,25 @@ internal object RadarSettings {
         getSharedPreferences(context).edit { putString(KEY_TRACKING_OPTIONS, optionsJson) }
     }
 
+    internal fun removeTrackingOptions(context: Context) {
```

Comment:
@bschoeneweis @corypisano 

unused.  Bug?  Or was just added for completeness.

Reply:
ah yes was being used earlier but no longer relevant, I can remove